exercises/concept/currency-exchange/.docs/instructions.md

```diff
@@ -7,12 +7,11 @@ Your friend Chandler plans to visit exotic countries all around the world. Sadly
 Create the `exchange_money()` function, taking 2 parameters:
 
 1. `budget` : The amount of money you are planning to exchange.
-2. `exchange_rate` : Unit value of the foreign currency.
+2. `exchange_rate` : The amount of domestic currency that equals one unit of foreign currency.
```

Comment:
Should this be?
```suggestion
2. `exchange_rate` : The amount of domestic currency that equals to one unit of foreign currency.
```
I think either of them are proper English, but I am not sure. What's your thought?

Reply:
Either the `that` or the `to` should go (I think) grammatically.  So one of these two (_emphasis mine_).  I agree that your phrasing (_so the second option_) sounds better:

> `exchange_rate` : The amount of domestic currency _**equal to**_ one unit of foreign currency.

OR

> `exchange_rate` : The amount of domestic currency **_that equals_** one unit of foreign currency.

Reply:
Third option:

> `exchange_rate` : The amount of domestic currency **_that is equal to_** one unit of foreign currency.

But that's super-formal, and feels a bit awkward to me.

Reply:
IMO, this might sound the best. We just need a fix right now, I honestly don't think we have to be stuck on a single sentence for something we are going to rewrite anyway. 
```suggestion
2. `exchange_rate` : The amount of domestic currency equal to one unit of foreign currency.
```